app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -253,7 +249,7 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
                 }
             } ?: finish()
         }
-        viewModelFactory.create(threadId, MessagingModuleConfiguration.shared.getUserED25519KeyPair(), contentResolver)
+        viewModelFactory.create(this@ConversationActivityV2, threadId, MessagingModuleConfiguration.shared.getUserED25519KeyPair(), contentResolver)
```

Comment:
I think `viewModels` survives config changes, so this will be a memory leak. Can you pass app context?

context is only used in `Recipient#getContactPhoto`... ideally we remove it.